File: 9/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        long y=0;
		int x_=x;
		while(x!=0) {
			y=y*10+x%10;
			x=x/10;
		}
		if(x_==y) return true;
		else return false;
    }
}

File: 9/Solution.c
bool isPalindrome(int x){
    if(x<0) return false;
    int x1=x;
    long x_=0;
    while(x1>0){
        x_ = x_*10+x1%10;
        x1 = x1/10;
    }
    if(x==x_) return true;
    else return false;
}